ci-operator/step-registry/aws/provision/iam-user/minimal-permission/aws-provision-iam-user-minimal-permission-chain.yaml

```diff
@@ -0,0 +1,7 @@
+chain:
+  as: aws-provision-iam-user-minimal-permission
+  steps:
+    - ref: ipi-conf-aws-user-min-permissions
+    - ref: aws-provision-iam-user
+  documentation: |-
+    Create an IAM user with minimum installer-required or/and ccoctl-required permissions.
```

Comment:
@jianlinliu @gpei @r4f4 Please review `aws-provision-iam-user-minimal-permission` chain again.
Now, `ipi-conf-aws-user-min-permissions` and `aws-provision-iam-user` support creating IAM users for both ccoctl and installer via flag `AWS_INSTALL_USE_MINIMAL_PERMISSIONS` and `AWS_CCOCTL_USE_MINIMAL_PERMISSIONS`, thanks